src/utils/worker-process.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import childProcess from 'child_process';
+import util from 'util';
+
+export interface ExecResult {
```

Comment:
this type is not used?

Reply:
I missed using it, i will update the code.